### DISCUSSION TITLE
Don't suggest people to follow. I just want release info for repos I follow

### DISCUSSION THREAD

**Discussion by horvatic** [community/community#13634] [Other Feature Feedback, Questions, & Ideas]
This isn't facebook, I want my feed to be:

1) Releases for repos I follow

Anything else isn't needed, as most people know what they need when on github. If I could have just release info then a bot could consume it, and auto update my code.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

in your case, just click the Repo and select only Releases

![image](https://user-images.githubusercontent.com/47092464/183014543-309669be-5429-4f86-9574-372699a48694.png)
